typescript/infra/config/environments/mainnet3/warp/addresses.json

```diff
@@ -1,9 +1,10 @@
 {
   "injective": {
+    "native": "inj1mv9tjvkaw7x8w8y9vds8pkfq46g2vcfkjehc6k",
     "router": "inj1mv9tjvkaw7x8w8y9vds8pkfq46g2vcfkjehc6k"
```

Comment:
Can you clarify why here there are both type + router keys but in the file below there are just type keys?

Reply:
Native is needed for artifact recovery for EVM deployment but not needed for cosmwasm. Just put it for symmetry.